frontend/src/features/env/queries.ts

```diff
@@ -5,11 +5,12 @@ import { ClientEnvVars } from '~shared/types/core'
 
 import { ApiError } from '~typings/core'
 
-import { getClientEnvVars, getSwitchEnvFormView } from './EnvService'
+import { getClientEnvVars, getFeedbackFormView } from './EnvService'
 
 const envKeys = {
   base: ['env'],
-  viewSwitchEnvForm: ['viewSwitchEnvForm'] as const,
+  adminFeedbackForm: ['adminFeedbackForm'] as const,
+  publicFeedbackForm: ['publicFeedbackForm'] as const,
```

Comment:
comment: feels like this whole feedback thing should not be in the `env` feature module and instead should be in it's own module. used to close one eye because was supposed to be temporary. is that still the case? if going to be stuck here forever we should move it to its own module.

out of scope of this PR though, we could check whether it's here to stay or not and create an issue for relocation otherwise.

Reply:
I agree, but for now kept it here since the future is a bit unclear. We also don't yet have clarity on whether we're reusing the feedback icon for anything else (e.g. enable feedback to admins in public forms, like you said).

Based on my convo with pearly, we are going to be deprecating the entire public feedback form soon, and only keep the admin feedback form, once we close off switchbacks to angular. Also currently, the feedback and switch env flows are separated for admins, but they are still together for respondents, so I decided not to separate them for now. We can rename and refactor a bit when we delete the respondent modal.